src/css/controls/imports/jwplayerlayout.less

```diff
@@ -17,7 +17,7 @@
     transition: opacity 250ms cubic-bezier(0, 0.25, 0.25, 1);
     z-index: 0;
 
-    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting) & {
+    .jwplayer.jw-state-playing.jw-flag-user-inactive:not(.jw-flag-casting):not(.jw-flag-ads) & {
```

Comment:
The `.jw-controlbar` element has its own transition state that's triggered when the `.jw-flag-user-inactive` is activated.